contracts/wallet.sol

```diff
@@ -238,279 +238,117 @@ contract AddressWhitelist is ControllableOwnable, SelfCallableOwnable {
-        return _gasTopUpLimit._getAvailableLimit();
-    }
-
-    /// @dev Is there an active gas top-up limit change
-    function gasTopUpLimitPending() external view returns (uint) {
-        return _gasTopUpLimit.pending;
-    }
-
-    /// @dev Has the gas top-up limit been initialised
-    function gasTopUpLimitUpdateable() external view returns (bool) {
-        return _gasTopUpLimit.updateable;
-    }
-
-    /// @dev View how much gas top-up has been spent already
-    function gasTopUpLimitValue() external view returns (uint) {
-        return _gasTopUpLimit.value;
-    }
-
-    /// @dev Sets the daily gas top up limit.
-    /// @param _amount is the gas top up amount in wei.
-    function setGasTopUpLimit(uint _amount) external onlyOwnerOrSelf {
-        require(_MINIMUM_GAS_TOPUP_LIMIT <= _amount && _amount <= _MAXIMUM_GAS_TOPUP_LIMIT, "out of range top-up");
-        _gasTopUpLimit._setLimit(_amount);
-        emit SetGasTopUpLimit(msg.sender, _amount);
-    }
-
-    /// @dev Submit a daily gas top up limit update.
-    /// @param _amount is the daily top up gas limit amount in wei.
-    function submitGasTopUpLimitUpdate(uint _amount) external onlyOwnerOrSelf {
-        require(_MINIMUM_GAS_TOPUP_LIMIT <= _amount && _amount <= _MAXIMUM_GAS_TOPUP_LIMIT, "out of range top-up");
-        _gasTopUpLimit._submitLimitUpdate(_amount);
-        emit SubmittedGasTopUpLimitUpdate(_amount);
-    }
-
-}
-
-
-/// @title LoadLimit provides daily load limit functionality.
-contract LoadLimit is ControllableOwnable, SelfCallableOwnable {
-
-    event SetLoadLimit(address _sender, uint _amount);
-    event SubmittedLoadLimitUpdate(uint _amount);
-
-    uint constant private _MINIMUM_LOAD_LIMIT = 1 finney;
-    uint private _maximumLoadLimit;
-
-    using DailyLimitTrait for DailyLimitTrait.DailyLimit;
-
-    DailyLimitTrait.DailyLimit internal _loadLimit;
-
-    /// @dev Sets a daily card load limit.
-    /// @param _amount is the card load amount in current stablecoin base units.
-    function setLoadLimit(uint _amount) external onlyOwnerOrSelf {
-        require(_MINIMUM_LOAD_LIMIT <= _amount && _amount <= _maximumLoadLimit, "out of range load amount");
-        _loadLimit._setLimit(_amount);
-        emit SetLoadLimit(msg.sender, _amount);
-    }
 
-    /// @dev Submit a daily load limit update.
-    /// @param _amount is the daily load limit amount in wei.
-    function submitLoadLimitUpdate(uint _amount) external onlyOwnerOrSelf {
-        require(_MINIMUM_LOAD_LIMIT <= _amount && _amount <= _maximumLoadLimit, "out of range load amount");
-        _loadLimit._submitLimitUpdate(_amount);
-        emit SubmittedLoadLimitUpdate(_amount);
+    /// @dev Use up amount within the daily limit. Will fail if amount is larger than available limit.
+    function _enforceDailyLimit(uint _amount) internal {
+        // Account for the limit daily reset.
+        _updateAvailableDailyLimit();
+        require(_available >= _amount, "available<amount");
```

Comment:
I'm not sure if using symbols in log messages is a good idea - it could cause problems when parsing (> < = ! ? etc.)

Reply:
Good catch, it was done to reduce contract size when we had issues, now we can afford to make the log messages a bit more expressive :)